<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>417</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>0</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>371</width>
     <height>311</height>
    </rect>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="AddTab">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Add tab</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>161</width>
     <height>80</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="tabsClosable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QListWidget" name="listClients">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSendMessage">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>380</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Отправить</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textMessage">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>buttonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddTab</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>addTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>tabCloseRequested(int)</signal>
   <receiver>Form</receiver>
   <slot>tabClosed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listClients</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>Form</receiver>
   <slot>listItemClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSendMessage</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>buttonClickSendMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClick()</slot>
  <slot>addTab()</slot>
  <slot>tabClosed()</slot>
  <slot>listItemClick()</slot>
  <slot>buttonClickSendMessage()</slot>
 </slots>
</ui>
